app/login/page.tsx

```diff
@@ -0,0 +1,71 @@
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { Card, CardHeader, CardContent, CardFooter, CardTitle, CardDescription } from "@/components/ui/card";
+import { Checkbox } from "@/components/ui/checkbox";
+import Link from "next/link";
+import Image from "next/image";
+
+const FormInput = ({ label, type, placeholder }) => (
+  <div>
+    <label className="block text-sm font-medium text-gray-700">{label}</label>
+    <Input type={type} placeholder={placeholder} className="w-full mt-1 rounded-full" />
+  </div>
+);
```

Comment:
convert to regular named function instead of an arrow function. Para lang standardized ang naming ng Components

Reply:
also di nag error sayo ang typescript? there should be type annotations for the parameters right?

something like `{ label, type, placeholder }: {label: string, type: string, placeholder: string}`

not sure lang pero doesnt look right nga walang type annotation